pkg/conf/config.go

```diff
@@ -35,3 +41,35 @@ func GetBuilderKey() (string, error) {
 	builderKey := string(builderKeyBytes)
 	return builderKey, nil
 }
+
+func GetStorageParams(env sys.Env) (Parameters, error) {
```

Comment:
I'd like this func to go into a top level storage package. I just merged #248, which creates the `github.com/deis/builder/pkg/storage` package, so that's a perfect place to put it